environments/development/analytical-platform/example-python-dag/dag.py

```diff
@@ -50,5 +51,12 @@
             secret=f"{PROJECT}-{WORKFLOW}-example",
             key="data"
         )
-    ]
+    ],
+    on_success_callback=[
+        send_slack_notification(
+            text="The task {{ ti.task_id }} succeeded",
+            channel="#analytical-platform-airflow-testing",
```

Comment:
```suggestion
            channel="analytical-platform-airflow-testing",
```
I'm not sure you need the `#` for this, its implied